packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -224,3 +233,77 @@ function Example() {
     </Flex>
   );
 }
+
+function Pending(props) {
+
+  return (
+    <>
+      <View backgroundColor={props.variant === 'overBackground' ? 'static-blue-700' : undefined} padding={16}>
+        <PendingButtonComponent {...props}>click me!</PendingButtonComponent>
+      </View>
+
+      <View backgroundColor={props.variant === 'overBackground' ? 'static-blue-700' : undefined} padding={16}>
+        <PendingButtonComponent
+          {...props}>
+          <Bell />
+          <Text>I have an icon</Text>
+        </PendingButtonComponent>
+      </View>
+
+      <View backgroundColor={props.variant === 'overBackground' ? 'static-blue-700' : undefined} padding={16}>
+        <PendingButtonOnClickComponent
+          {...props}>
+          <Text>with onClick</Text>
+        </PendingButtonOnClickComponent>
+      </View>
+
+      <View padding={16}>
+        <Button variant="primary" isPending={props.isPending} onPress={() => {window.alert('use storybook control to change this button isPending prop');}}>
+          <Text>Controlled pending</Text>
+        </Button>
+      </View>
+    </>
+  );
+}
+let timerValue = 5000;
+function PendingButtonComponent(props) {
+  let [isPending, setPending] = useState(false);
+
+  let handlePress = (e) => {
+    action('press')(e);
+    setPending(true);
+    setTimeout(() => {
+      setPending(false);
+    }, timerValue);
+  };
+
+  return (
+    <Button
+      {...props}
+      isPending={isPending}
+      onPress={handlePress}>
+      {props.children}
+    </Button>
+  );
+}
+
+function PendingButtonOnClickComponent(props) {
+  let [isPending, setPending] = useState(false);
+
+  let handlePress = (e) => {
+    action('press')(e);
+    setPending(true);
+    setTimeout(() => {
+      setPending(false);
+    }, timerValue);
+  };
```

Comment:
I think you are correct that this is a problem is unrelated to my changes. Also, since `onClick` is deprecated, I'm inclined not to worry much about it. `Press` events work fine with keyboard. How do others on the team feel?

Reply:
It never has, onClick is specific to mouse, it never handled keyboard. The press events are the correct way to go.